src/core/server/saved_objects/service/lib/utils.ts

```diff
@@ -80,4 +80,11 @@ export class SavedObjectsUtils {
     total: 0,
     saved_objects: [],
   });
+
+  public static filterWorkspacesAccordingToBaseWorkspaces(
+    targetWorkspaces?: string[],
+    baseWorkspaces?: string[]
```

Comment:
Nit: renaming to `sourceWorkspace` could be better.

Reply:
Good advice.

Reply:
Done.